src/Public.Api/Public.Api.csproj

```diff
@@ -44,6 +45,8 @@
     <Content Include="Dockerfile" CopyToOutputDirectory="Always" />
     <Content Include="appsettings.json" CopyToOutputDirectory="Always" />
     <Content Include="appsettings.*.json" CopyToOutputDirectory="Always" />
+    <None Remove="appsettings - Copy.json" />
```

Comment:
remove these 2 lines

Reply:
2 lines are still there -- remove appsettings - Copy.json and appsettings.laptop-g7fahqcf.json